### PR TITLE
Replace generic params with sensible names in routes

### PR DESCRIPTION
**Pull request by aaronxsu**
This PR:
- makes `spec.yml` and `spec/spec.yml` in line with each other
- replaces generic params with sensible names in routes

Closes https://github.com/raster-foundry/raster-foundry-python-client/issues/54

### CODE REVIEW

File: spec.yml [hunk 1]
```diff
@@ -213,14 +213,14 @@ paths:
             maxItems: 5
             items:
               $ref: '#/definitions/UserSearched'
-  /users/{uuid}/:
```
Comment (jisantuc) [inline spec.yml:216]: Lol at calling this a `uuid` when it's not even. Generic _and_ wrong :slot_machine:

File: spec.yml [hunk 2]
```diff
@@ -3607,14 +3652,91 @@ parameters:
...
     type: number
     format: int32
-  uuid:
-    name: uuid
+  refreshTokenID:
+    name: refreshTokenID
+    in: path
+    required: true
+    type: string
+  datasourceID:
```
Comment (jisantuc) [inline spec.yml:3660]: Is it possible to fill these in with a `ref` to a generic `uuidPathParam` for all the `in: path`, `required: true`, `type: string`, `format: uuid` pieces, and then just provide the name?

**Review by jisantuc — COMMENTED** [review]
synchronization looks good:

```
james@rf-dev-env-eu ~/raster-foundry-api-spec> diff spec.yml spec/spec.yml 
james@rf-dev-env-eu ~/raster-foundry-api-spec>
```

just one question about possibly more efficient representation of all of the uuid params we have that have 4/6 lines the same

File: spec.yml (hunk 2 above)
Comment (aaronxsu) [reply spec.yml:3660]: @jisantuc lemme try

**Review by jisantuc — APPROVED** [review]

File: spec.yml (hunk 2 above)
Comment (aaronxsu) [reply spec.yml:3660]: yea, it does not seem like this is possible -- though it could be less repetitive if it was.

**Pull request merged**